docs/content/topics/contributor-guide/reviewing.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Code Review Guide
+description: The Brigade guide to reviewing PRs
+section: contributor-guide
+weight: 6
+---
+
+This document outlines the Brigade code review process that is applied to all
+contributions (pull requests) and covers things like who can review PRs, when
+and _how_.
+
+> ⚠️&nbsp;&nbsp;Throughout this document, "project" (with a lowercase "p") is
+> used to denote individual software projects or source code repositories that
+> are related in some way to Brigade and owned by the
+> [@brigadecore](https://github.com/brigadecore) GitHub org. In contrast,
+> "Project" (with an uppercase "P") is used to denote the entire breadth of
+> Brigade as a "program" encompassing Brigade itself and _all_ related software
+> projects or source code repositories owned by the __@brigadecore__ org.
+
+## Reviewers
+
+For the Brigade Project, __Reviewer__ is not a formal role in the way that a
+__Project Maintainer__ or __Core Maintainer__ is. Here, a Reviewer is anyone
+who reviews a PR, whether voluntarily or to fulfill commitments they have to the
```

Comment:
"whether voluntarily or to fulfill commitments they have" that sounds a bit off because I sure hope all interactions with the project are voluntary! Did you mean whether it's a review done in an official capacity?

Reply:
I was thinking that maintainers essentially, by virtue of their role, have a duty to perform code reviews. That was all.

Happy to reword this if you think there's a better way to say it.

Reply:
It's up to you but I'd suggest not using the word voluntary since its the wrong word for what you are talking about, and makes it sound draconian. 😀

Reply:
I agree it sounds weird. I'll find a way to fix it.